source/hotkeys_.d

```diff
@@ -209,7 +213,7 @@ struct ModHotkey{
 
 struct Hotkeys{
 	bool capsIsCtrl=true;
-	int[] moveForward,moveBackward,turnLeft,turnRight;
+	int[] moveForward,moveBackward,turnLeft,turnRight, moveUp, moveDown;
```

Comment:
(Minor: inconsistent whitespace.)

Reply:
obviously, can't make oyu introduce whitespaces but i could recommend a github action to (re)format the code.

Reply:
moved. same for camera movements too.